docs/get_started.md

```diff
@@ -113,66 +123,108 @@ the creation of a new management pack project.
 ???+ info
 
      For complete documentation of the `mp-init` tool, including an overview of its output,
-     see the [MP Initialization Tool Documentation](references/mp-init.md).
+     see the [MP Initialization Tool Documentation](references/mp-init.md). 
+     For Java projects documentation consult [Java project](references/python_project/index.md) section   
+     For Python projects documentation consult [Python project](references/java_project/index.md) section   
 
 ### Project Templates
-Both templates create a file system that has the basic project structure required to develop and build a Management Pack.
-Each file and directory is discussed in depth in the [mp-init](references/mp-init.md) documentation. `app/adapter.py` is the adapter's
-entry point and the best starting point.
+All the project templates create a file system that has the basic project structure required to develop and build a Management Pack.
+Each file and directory is discussed in depth in the [mp-init](references/mp-init.md) documentation.
+To learn more about the specifics of each language: 
+    - Python 
+    - Java 
```

Comment:
This seems unfinished, and possibly redundant. Are the files going to be discussed in `mp-init`, or in a language-specific place?